specs/pvs/corla.pvs

```diff
@@ -1043,14 +1098,18 @@ END cast_vote_records
 
 %+BEGIN ui
 
-%%% \section{User Interfaces}
+%%% \subsection{User Interfaces}
 
 %%% The user interfaces of the system are the visible interactive
 %%% parts of the application. There are three different user
 %%% interfaces in the RLA Tool. The precise names of these interfaces
 %%% are still under discussion; one is for CDOS personnel responsible
 %%% for the audit at the state-level, one is for county personnel at
 %%% the county-level, and one is for audit board members.
+%%% Note that all these user interfaces both receive and 
+%%% provide information.
+%%% Because the public dashboard only pushes information out, 
+%%% we do not consider it a "user interface".
```

Comment:
I agree with you @nealmcb.  A UI is a UI, regardless of if it is read-only or not. We need to specify events and scenarios even for the public dashboard, in this regards.